packages/legacy/core/App/components/misc/InactivityWrapper.tsx

```diff
@@ -0,0 +1,121 @@
+import { useAgent } from '@credo-ts/react-hooks'
+import React, { PropsWithChildren, useEffect, useRef } from 'react'
+import { AppState, PanResponder, View } from 'react-native'
+import { useAuth } from '../../contexts/auth'
+import { useStore } from '../../contexts/store'
+import { DispatchAction } from '../../contexts/reducers/store'
+import { TOKENS, useServices } from '../../container-api'
+
+export enum LockOutTime {
```

Comment:
Enums are somewhat of a hack in typescript. I think we favour `as const` like

```
const X = {
  Blah: 'zzzzzz',
  Blarb: 'rrrrr',
} as const
```

Reply:
Happy to change that. Is there a way to still have this object act as a type when defining something like a type or interface? for example:

```typescript
const MyConst = {
  Blah: 'something'
} as const

type MyType = {
  someAttribute: MyConst
}

```